.github/ISSUE_TEMPLATE/minor.md

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
What's the use case for `minor` and `major?

Reply:
I found that sometimes that larger updates would need to be made to existing features sometimes (this would happen when a learning lab is sent to a mentor/point of contact for review and they don't like something about it)...with our new learning lab structure hopefully such situations do not occur but I added it in just in case -- if you feel that it's redundant (e.g. maybe the `feature` issue can take care of it?) then I'm down to remove

Reply:
I think generally changing something would still either be a feature change or a refactor? This is how they do it in larger repositories + just using a small tag or something like that